graph/graph.go

```diff
@@ -103,8 +103,10 @@ func newChart(series []chart.Series, markers []chart.GridLine, width, height int
 	}
 	graph.Elements = []chart.Renderable{
 		legend(&graph, chart.Style{
-			FillColor:   drawing.Color{A: 100},
-			FontColor:   chart.ColorWhite,
+			FillColor: drawing.Color{A: 100},
+			FontStyle: chart.FontStyle{
```

Comment:
Font properties have been now grouped together into a `FontStyle` struct.  This is to make it easier to set the same font properties to multiple style elements.